src/providers/execution/contracts/lido.py

```diff
@@ -36,6 +37,67 @@ def handle_oracle_report(
         while passing empty `_withdrawalFinalizationBatches` and `_simulatedShareRate` == 0, plugging the returned values
         to the following formula: `_simulatedShareRate = (postTotalPooledEther * 1e27) / postTotalShares`
         """
+        try:
+            return self._handle_oracle_report(
+                timestamp,
+                time_elapsed,
+                validators_count,
+                cl_balance,
+                withdrawal_vault_balance,
+                el_rewards,
+                shares_to_burn,
+                accounting_oracle_address,
+                HexStr('0x' + ref_slot.to_bytes(32).hex()),
+                block_identifier,
+            )
+        except ValueError as error:
+            # {'code': -32602, 'message': 'invalid argument 2: hex number with leading zero digits'}
```

Comment:
could we check exact message here? in current implementation we will always try to use Erigon format, but the error might be not about that

Reply:
I'm afraid to rely on the node response, I would just guess if it doesn't work try another format